HOWTO.md

```diff
@@ -14,9 +14,15 @@ resources for you.
 The remainder of this document explains how to write SAM templates and 
 deploy them via AWS CloudFormation. 
 
-## Writing SAM Template
+## Getting started with SAM Template
 Check out the [latest specification](versions/2016-10-31.md) for details on how to write a SAM template
 
+
+You could also use the [aws-sam-cli](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-publishing-applications.html) to get started 
```

Comment:
As a GitHub readme file, it's probably more direct to link to the GitHub repo for SAM CLI:
https://github.com/awslabs/aws-sam-cli